src/hooks/useTogglePointerEvents.js

```diff
@@ -0,0 +1,60 @@
+import { useEffect } from "react";
+
+import { MOUSE_EVENTS, KEYS, NON_NAVIGATION_KEYS } from "src/constants";
+
+/**
+ * This hook is responsible to disable the mouse when the user
+ * starts to navigate using the keyboard and enables the mouse
+ * and the user moves it
+ */
+export const useTogglePointerEvents = () => {
+  useEffect(() => {
+    if (typeof window !== "undefined" && typeof document !== "undefined") {
+      const eventListener = event =>
+        Object.values(KEYS).includes(event.key) &&
+        !NON_NAVIGATION_KEYS.includes(event.key) &&
+        togglePointerEvents(MOUSE_EVENTS.KEY_PRESS);
+
+      window.addEventListener("keydown", eventListener);
+      return () => {
+        window.removeEventListener("keydown", eventListener);
+        window.removeEventListener("mousemove", mouseMoveListener);
+      };
+    }
+  }, []);
+};
+
+/**
+ * Listener should be triggered once and then cleared itself
+ * It sets _document.body.style.pointerEvents_ to '' if it
+ * has changed it's value
+ */
+const mouseMoveListener = () =>
+  togglePointerEvents(MOUSE_EVENTS.MOUSE_MOVE) &&
+  window.removeEventListener("mousemove", mouseMoveListener);
```

Comment:
I'm pretty sure the `removeEventListener` never gets called because `togglePointerEvents` never gets called.

Something like this might work better:
```suggestion
const mouseMoveListener = () => {
  togglePointerEvents(MOUSE_EVENTS.MOUSE_MOVE);
  window.removeEventListener("mousemove", mouseMoveListener);
};
```